src/main/resources/com/redhat/jenkins/plugins/cachet/SourceTemplate/config.jelly

```diff
@@ -0,0 +1,12 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry title="${%Cachet API URL}" field="cachetUrl">
+        <f:textbox/>
+    </f:entry>
+    <f:entry title="${%Label}" field="label">
```

Comment:
Can you please add some documentation to describe what a label is and how it is used?

Reply:
I've added it [here](https://github.com/jenkinsci/cachet-gating-plugin/blob/de561f7a8d21488c03ec2ea30cabae76f3d72cd3/src/main/resources/com/redhat/jenkins/plugins/cachet/GlobalCachetConfiguration/help-label.html)